2025 Rules

RealOrangeOne opened the pull request:
## To do

- [x] Pull in robot design rules from SR rulebook
- [x] Pull in safety regs from SR rulebook
- [x] Smooth angles on line follow guide
- [x] Remove mentions of smallpeice / sponsors
- [ ] Confirm competition structure

RealOrangeOne committed:
Simplify can penalty

RealOrangeOne committed:
Update year

RealOrangeOne committed:
Remove mentions of smallpeice

RealOrangeOne committed:
Overhaul safety and design rules

Mostly copied from SR rulesbook

RealOrangeOne committed:
Smooth corners on line follow guide

prophile commented on src/index.html:
```diff
@@ -64,64 +53,111 @@ <h2><a class="anchor" id="game-rules"></a>Game Rules</h2>
-            <li>
-              The power board must be mounted on the top of the robot, with its buttons easily accessible.
-          </ol>
+          During each match, robots must fit within a 600 mm × 600 mm horizontal square at all times. The robot's height is not restricted during the match.
```
Issue on the height front: what do we do if a robot falls over? This was spotted at SR2025.

RealOrangeOne replied:
Given this game, I'm going to just remove this point. Let's just keep a maximum size throughout.

RealOrangeOne committed:
Enforce size constraint at all times

RealOrangeOne committed:
Mention included battery box

RealOrangeOne committed:
Add additional scoring line

RealOrangeOne committed:
De-optimise line follow guide

It was a little too optimal

RealOrangeOne committed:
Update actions

RealOrangeOne committed:
Add background to scoring line legend

RealOrangeOne committed:
Add background to line follow guide legend

RealOrangeOne committed:
Improve wording on cans being knocked over

inventor02 committed:
Add wrangler config

RealOrangeOne committed:
Drop safety exemptions

RealOrangeOne committed:
Add measurements to top right can

RealOrangeOne committed:
Improve safety wording

RealOrangeOne committed:
Move tidy wiring to design rule

RealOrangeOne committed:
Move flag mount section to specifications

RealOrangeOne committed:
Remove 50V rule

There's no way they'll get that high.

This commit absolves the author of any liability which comes from damage
caused by power levels previously excluded by the 50V limit.
